gulp/tasks/lint.js

```diff
@@ -1,21 +1,21 @@
 (function() {
     'use strict';
 
-    var jshint = require('gulp-jshint'),
+    var eslint = require('gulp-eslint'),
         gulp = require('gulp'),
         paths = {
             lint: [
```

Comment:
Do we need both this and the .eslintignore? Maybe this can just be ``**/*.js`` and let the exceptions be handled in the ignore file.